go/tools/builders/nogo_main.go

```diff
@@ -90,6 +91,9 @@ func run(args []string) error {
 		return fmt.Errorf("errors found by nogo during build-time code analysis:\n%s\n", diagnostics)
 	}
 	if *xPath != "" {
+		if runtime.GOOS != "darwin" {
```

Comment:
Why not `abs` on darwin?

We avoid changing paths for executables and shared libraries on darwin because they get baked into the binary and affect loading. We're just writing files here though, so it should be fine.

Reply:
Honestly, I wasn't sure what exactly these files did, and I thought seeing some .a files on the command line (I may have been mistaken), that's why. If you think abs() is fine on any platform, so be it. (Maybe I'll even find time to test it on a darwin machine later today)